product/reports/170_Configuration Management - Containers/100_Pod counts For Container Images by Project.yaml

```diff
@@ -0,0 +1,63 @@
+---
+title: Pod counts For Container Images by Project
+rpt_group: Custom
+rpt_type: Custom
+priority:
+db: ContainerProject
+cols:
+- name
+include:
+  container_images:
+    columns:
+    - name
+  container_groups:
+    columns:
+    - name
+col_order:
+- name
+- container_images.name
+- container_groups.name
+headers:
+- Project Name
+- Image Name
+- Pod Name
+conditions: !ruby/object:MiqExpression
+  exp:
+    ">":
+      field: ContainerProject-images_count
+      value: '0'
+  context_type:
+order: Ascending
+sortby:
+- name
+group: c
+graph:
+dims:
+filename:
+file_mtime:
+categories: []
+timeline:
+template_type: report
+where_clause:
+db_options: {}
+generate_cols:
+generate_rows:
+col_formats:
+-
+-
+-
+tz:
+time_profile_id:
+display_filter:
+col_options:
+  name:
+    :break_label: 'Name: '
+rpt_options:
+  :pdf:
+    :page_size: US-Letter
+  :queue_timeout:
+  :summary:
+    :hide_detail_rows: false
+miq_group_id: 2
+user_id: 1
```

Comment:
Thanks @zeari, you're right... you'll see that the merge commit, https://github.com/ManageIQ/manageiq/commit/4a9a81e02167be2a32f8fc2c8777d82eb617863d, doesn't show the change to the seeding_spec.rb to bump the number from 137 to 139 because 87cb2104b4494b0fcfc3ed1afd6d13dca79a6d91 already did it.  I'll open a PR to fix it.